### ISSUE TITLE
French accessibility Voiceover

### ISSUE THREAD

**Issue by Rajat-Sen** [StatCan/StatcanPublicationUAT#223]
When the user switches to French language, the voiceover is reading in English instead of French.

**Comment by ForumG-SC** [comment]
Which screen?

**Comment by goodyke** [comment]
Rajat could you please have a look at this bug to see if it is a true bug or simply that the Mobility team did not have the correct settings when using voiceover?  The Jira ticket associated with this is QLTY-136

**Comment by Rajat-Sen** [comment]
I looked into this and I think it is functioning properly.

The reason they would be having this issue due to:
Device language is in English and the app language in French. By changing the language of app, we cannot change the system language settings.

Kerry, would you be able to ask them if they have faced any issues if both the device language and the app language are same. Until we hear back, I am closing this ticket for now as, it is not a bug.

**Comment by goodyke** [comment]
@Rajat-Sen the only error they reported was when the device language and the app language was not the same... therefore the associated Jira ticket QLTY-136 has been set to Done with a comment explaining that the device language setting must be set to match the display app language setting when using a screen reader.
